docs/src/pages/upgrade/upgrade.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: "../../layouts/docs/Layout.astro"
+title: "Upgrade"
+index: 1
+---
+
+# Installing kairos on baremetal, VMs, ...
```

Comment:
Same here, this shouldn't have a section with just few sentences - the old docs have a different format that doesn't fit here